setup.sh

```diff
@@ -47,7 +47,7 @@ else
    fi
 fi
 
-pip install django==1.10.6 pillow grandalf django-prepared-query appdirs
+pip install django pillow grandalf django-prepared-query appdirs
```

Comment:
I would rather keep some version info here, so that others get a out-of-box config that we have somewhat tested... Perhaps just major and minor versions, not micro ?

Reply:
Good practice. At some point, I may take a look at making the overall packaging a bit more standard, if that's ok. For now, I changed it to `pip ... django>=1.10 ...`. Does that work for you?

Reply:
Please go ahead. I am fluent with python itself, but not so familiar with the typical layout of projects, so that's something that would take me a long time most likely.

I would go ahead and require django 2.1 or whatever version you tested